src/resources/Correction/GeneralPartnershipResource.ts

```diff
@@ -15,6 +15,8 @@ export const GeneralPartnershipResource: any = {
     nameChangeOptions: [
       CorrectionTypes.CORRECT_NEW_NR
     ],
+    typeChangeInfo: 'You cannot change the business type of a Sole Proprietorship / DBA. You must form a new ' +
```

Comment:
This is the resource file for a GP so I think this is the wrong text.

Reply:
This is for the tooltip.

Reply:
It's now fixed.